README.md

```diff
@@ -34,18 +36,48 @@ OpenCensus collects distributed tracing. It is able to do so by propagating span
 
 OpenCensus is vendor-agnostic and can upload data to any backend with various exporter implementations. Even though, OpenCensus provides support for many backends, users can also implement their own exporters for proprietary and unofficially supported backends. Currently, OpenCensus supports:
 
+#### Trace exporters
 - [Stackdriver](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-exporter-stackdriver/README.md)
-- [Zipkin](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-exporter-zipkin/README.md)
-- [Z-Pages](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-exporter-zpages/README.md)
 - [Jaeger](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-exporter-jaeger/README.md)
+- [Zipkin](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-exporter-zipkin/README.md)
 - [Instana](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-exporter-instana/README.md)
 
+#### Stats/Metrics exporters
+- [Stackdriver](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-exporter-stackdriver/README.md)
+- [Prometheus](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-exporter-prometheus/README.md)
+
+#### How to setup debugging Z-Pages?
+-  If the application owner wants to export in-process tracing and stats data via HTML debugging pages see this [Z-Pages](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-exporter-zpages/README.md).
+
 If no exporter is registered in the tracing instance, as default, a console log exporter is used.
 
+## Versioning
+
+This library follows [Semantic Versioning][semver].
+
```

Comment:
Is `opencensus-node` is actually at an Alpha stage? If so, should that be included in the list of stages?